example-card-integration/data-connector/actions/importaction.ts

```diff
@@ -0,0 +1,39 @@
+import {DataConnectorAsynchronousAction} from 'lucid-extension-sdk';
+import {CollectionName, DataSourceName} from '../../common/names';
+import {ExternalExampleDataClient} from '../../data/exampleexternaldataclient';
+import {exampleCollectionSchema, ExampleTaskType} from '../schema/examplecollectionschema';
+import {fullTaskDataToLucidFormat} from '../schema/exampletaskutil';
+
+export const importAction: (action: DataConnectorAsynchronousAction) => Promise<{success: boolean}> = async (
+    action,
+) => {
+    const primaryKeys = (action.data as any).taskIds as string[];
+    const itemsToAdd: ExampleTaskType[] = getTaskData(primaryKeys).filter((item) =>
+        primaryKeys.some((key) => JSON.stringify(item.id) === key),
+    );
+
+    action.client.update({
```

Comment:
Nit: I think we should use `declareSchema` in here so people know they can use that (it also provides the `forItem` helper for primary keys)

Reply:
exampleCollectionSchema uses declareSchema